app/cards/[id]/page.tsx

```diff
@@ -1,19 +1,71 @@
 "use client";
 import Link from "next/link";
+import { useRouter } from "next/navigation";
 import { useParams } from "next/navigation";
 import { Button } from "@/components/uikit/button";
+import { showCard } from "@/domains/cards/services";
+import { Input } from "../../../components/uikit/input";
+import { deleteCard } from "@/domains/cards/services";
 
 export default function CardId() {
   const params = useParams();
```

Comment:
```suggestion
  const { id } = useParams();
```

І тоді можна видалити:
`const id = params.id;`